lib/lita/handlers/reviewme.rb

```diff
@@ -96,6 +96,7 @@ def comment_on_github(response, room: get_room(response))
         rescue Octokit::Error
           response.reply("Unable to check who issued the pull request. Sorry if you end up being assigned your own PR!")
         end
+        return response.reply('Sorry, no reviewers found') unless reviewer
```

Comment:
I guess you opted to return at this point rather than earlier to cover the case where there is only 1 reviewer and it's _you_? I'd probably rather make that case more explicit, but I don't think that's your job 🙂